tests/kernel/pipe/pipe_api/src/concurrency.c

```diff
@@ -29,15 +29,15 @@ static void thread_reset(void *arg1, void *arg2, void *arg3)
 
 static void thread_write(void *arg1, void *arg2, void *arg3)
 {
-	uint8_t garbage[dummy_data_size];
+	uint8_t garbage[DUMMY_DATA_SIZE] = {};
```

Comment:
FWIW this line changes semantics: it used to be garbage, now it's zero-filled.

Reply:
ok, but does that matter in this case?

Reply:
Probably not?  But in a patch to fix a warning it's generally best practice to change nothing else.